src/FileSystem-Tests-Core/FileSystemTest.class.st

```diff
@@ -679,6 +679,23 @@ FileSystemTest >> testRootIsNotAFile [
 	self deny: (filesystem isFile: Path root)
 ]
 
+{ #category : #'tests-streams-compatibility' }
+FileSystemTest >> testTooManyDirectoryDescriptorOpened [
+	| opened testSize |
+
+	self halt.
+	testSize := 1025.
+	opened := OrderedCollection new:testSize.
```

Comment:
Here giving the test size is not necessary :)

Reply:
It's just to be able to tests several sizes more easily change it, given that the number of open Directory Descriptor varies on each OS. But i'll remove it.

Reply:
I meant in the collection's initial size :).